### ISSUE TITLE
testing result

### ISSUE THREAD

**Issue by door5719** [qijiezhao/CFENet#13]
Hi sir ,I run your project with coco2014_val ,on win10 ,python3.5 ,pytorch0.41 ,
no bug,  and got the result like this: 

C:\Users\............\Anaconda3\python.exe D:/Python/CFENet/CFENet-working/test.py
Finished loading model!
loading annotations into memory...
Done (t=0.79s)
creating index...
index created!
minival2014 gt roidb loaded from D:\coco\coco_cache\val2014_gt_roidb.pkl
=> Total 5000 images to test.
  0%|          | 0/5000 [00:00<?, ?it/s]Begin to evaluate
100%|██████████| 5000/5000 [05:27<00:00, 16.27it/s]
0it [00:00, ?it/s]Evaluating detections
Collecting Results......
81it [00:04, 18.96it/s]
Writing results json to eval\COCO\detections_minival2014_results.json
loading annotations into memory...
Done (t=0.59s)
creating index...
index created!
Loading and preparing results...
DONE (t=0.82s)
creating index...
index created!
Running per image evaluation...
useSegm (deprecated) is not None. Running bbox evaluation
Evaluate annotation type *bbox*
DONE (t=20.16s).
Accumulating evaluation results...
DONE (t=2.45s).
~~~~ Mean and per-category AP @ IoU=[0.50,0.95] ~~~~
30.4
[39.805515252305426, 22.46525010297838, 26.507262921399732, 35.044814200025634, 58.42548938600688, 56.51351931396638, 61.16247328083879, 27.4913108863114, 16.274161628973722, 12.824321562301433, 53.85063919935348, 51.97903626969791, 32.08775069122522, 16.990110250353695, 22.81007802277641, 59.37883689563789, 54.55814590904055, 46.92329364332628, 38.328943150917645, 37.41084690805071, 52.08346436113952, 66.01604148444913, 53.39185296600959, 52.6905841689817, 7.743878493290978, 27.89868566174553, 6.079698879061883, 17.94021614465387, 24.815709212505357, 46.04429167730119, 14.785200237042014, 19.799667194841224, 24.02812665796755, 24.73045612394421, 15.667318805827946, 26.12754043063172, 38.18776250963001, 25.80383015005366, 32.37634388285044, 18.35351844001476, 17.845230299585126, 25.231800291244006, 16.852035653854287, 5.969282424144782, 5.5881263296010255, 30.403990739703186, 15.2340613698474, 11.497076902476278, 29.880159301790172, 22.18183660559775, 18.31879802168003, 13.695638366002663, 25.476639213213925, 40.88122143275187, 34.965163181539914, 29.902558475430823, 18.534251670132747, 38.42602708857324, 17.54745023720155, 37.65120946065206, 23.242469206401363, 53.80350809433423, 46.996335863056075, 48.98849783237312, 45.06427272602526, 12.941750976188976, 39.90299335913138, 24.42649917514803, 50.882576332768735, 30.335098317983224, 24.633663366336627, 28.51237604722684, 43.6472054205158, 6.717747942725008, 37.26465323607326, 23.372962816716548, 23.753231379733045, 35.09583453769022, 0.0, 8.72949791536005]
~~~~ Summary metrics ~~~~
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.304
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.491
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.319
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.122
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.331
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.466
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.260
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.380
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.396
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.162
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.436
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.592
Wrote COCO eval results to: eval\COCO\detection_results.pkl
Detect time per image: 0.039s
Nms time per image: 0.004s
Total time per image: 0.043s
FPS: 23.410 fps

This code repo paper: CFENet An Accurate and Efficient Single-Shot Object Detector for Autonomous Driving ?
This is a nice paper and I want to know when will the demo.py come. 
Thanks for your reading.

**Comment by door5719** [comment]
And the loss function is same with SSD?

**Comment by qijiezhao** [comment]
Both the speed and the accuracy are lower than mine. 
The suggested hardware configuration is: CUDA9.0+, cuDNN7.1.4+, pytorch0.4.1, NVIDIA-version 390.77+.
Besides this, your accuracy has 0.7 points lower than 31.1(minival).

**Comment by qijiezhao** [comment]
> And the loss function is same with SSD?

Yeah

**Comment by door5719** [comment]
I7-7700,32G RAM
GTX-1080
Cuda compilation tools, release 9.1, V9.1.85
